src/AccessibilityInsights.Extensions.AzureDevOps/LoadingWindow.xaml.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.VisualStudio.Services.Client.Controls;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+
+namespace AccessibilityInsights.Extensions.AzureDevOps
+{
+    /// <summary>
+    /// Interaction logic for LoadingWindow.xaml
+    /// </summary>
+    public partial class LoadingWindow : Window
+    {
+        private Func<Task> Operation { get; set; }
+
+        public LoadingWindow(Window owner, Func<Task> operation)
+        {
+            InitializeComponent();
+            this.Owner = owner;
+            this.Operation = operation;
+        }
+
+        /* ADO window from HandleLoginAsync throws if this window is closed while the other is up
```

Comment:
If this LoadingWindow is closed while the ADO interactive dialog is up, we get an unhandled exception (I think raised from HandleLoginAsync) that crashes the app